README.md

```diff
@@ -9,21 +9,13 @@ A Terraform Module to configure the Lacework Agentless Scanner.
 
 ## Requirements
 
-| Name | Version |
-|------|---------|
+| Name | Version   |
+|------|-----------|
 | <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.15.0 |
-| <a name="requirement_aws"></a> [aws](#requirement\_aws) | ~> 4.0 |
-| <a name="requirement_lacework"></a> [lacework](#requirement\_lacework) | ~> 1.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 4.0 |
 | <a name="requirement_random"></a> [random](#requirement\_random) | >= 2.1 |
-
-## Providers
-
-| Name | Version |
-|------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | ~> 4.0 |
-| <a name="provider_lacework"></a> [lacework](#provider\_lacework) | ~> 1.0 |
+| <a name="provider_lacework"></a> [lacework](#provider\_lacework) | ~> 1.8 |
 | <a name="provider_null"></a> [null](#provider\_null) | n/a |
```

Comment:
nit: Not related to this change but this is not needed as well.

Reply:
The null provider is being used by the following resource: `resource "null_resource" "check_organization_requires_global_input"`.
My suggestion is to add a minimum version. Because it's difficult to do this afterwards, I'd use the version which was the latest when this project had its first release.